Article/Article.js

```diff
@@ -1,19 +1,25 @@
-// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.domElement = domElement;
+    this.article = this.domElement.querySelector('.article');
     
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
+    // create a reference to the ".expandButton" class. 
+    this.expandButton = this.domElement.querySelector('.expandButton');
+    // console.log(this.domElement);
     
-  }
+    // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton.innerHTML = 'expand';
```

Comment:
try to stay away from `innerHTML` whenever possible due to potential security implications -- `innerText` would work just as well here, for example